server/webhooks/registration-parser.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+import 'babel-register';
+import _ from 'lodash'
+
+class cashnetJSON {
+    constructor(cnj) {
+        let { itemcnt } = cnj;
+        let refs = this.refs(cnj);
+        this.batchNumber = cnj['batchno'];
+        this.primaryContact = {
+            "email": cnj['MA3-PUEMAIL'],
+            "name": cnj['MA3-PUNAME']
+        };
+        this.participants = this.getParticipants(refs);
+        this.tshirts = this.getTshirts(refs);
+        this.transaction = cnj['tx'];
+        this.totalAmount = this.getTotalAmount(refs);
+    }
+    refs(cnj) {
+        let { itemcnt } = cnj;
+        let refArray = [];
+        _.forEach(_.range(1, _.toInteger(itemcnt) + 1), function (index) {
+            refArray.push({});
+            refArray[index - 1]['amount'] = cnj[`amount${index}`];
```

Comment:
Object keys that are not dynamic or requiring strings should just use dot notation
i.e. `SomeObject.thisKey`